src/octopeer-github/main/Database/EventObject/MousePositionEvent.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Created by Maarten on 26-05-2016.
+ */
+
+/**
+ * An MousePositionEvent contains the data that should be posted to a Database.
+ */
+interface MousePositionEvent {
+    /** position_x and position_y specify the position on the page (i.e. relative to the top-left corner of the document). */
```

Comment:
I'm not really a fan of inline comments. Especially not in such small interfaces. Consider moving the comments to the method TSDoc (in all EventObject interfaces)